library/Fission/Web/User/Password/Reset.hs

```diff
@@ -0,0 +1,31 @@
+{-# LANGUAGE MonoLocalBinds    #-}
+
+module Fission.Web.User.Password.Reset
+  ( API
+  , server
+  ) where
+
+import           RIO
+
+import           Database.Selda as Selda
+
+import           Servant
+
+import           Fission.Web.Server
+import qualified Fission.Web.Error    as Web.Err
+
+import           Fission.User                 as User
+import qualified Fission.User.Password.Types  as User
+
+type API = ReqBody '[JSON] User.Password
+        :> Post '[JSON] User.Password
```

Comment:
Per REST, this should be a `PUT`. I wish that they had named it better; `PUT` vs `POST` is confusing.

Reply:
Yup you're right. I often mix those up